### PR TITLE
Fix can_create_window.patch

### PR DESCRIPTION
**Pull request by codebytere**
Patch applies cleanly once more ✨ 

For https://github.com/electron/libchromiumcontent/pull/499.

/cc @alexeykuzmin

### CODE REVIEW

File: patches/common/chromium/can_create_window.patch [hunk 1]
```diff
@@ -23,13 +23,13 @@ index 1bf37e107a90..82ab04e1019d 100644
 +        }
 +        case storage::DataElement::TYPE_FILE: {
 +          element.SetToFilePathRange(iter->path, iter->offset, iter->length,
-+                                     iter->expected_modification_time);
++                                      iter->expected_modification_time);
```
Comment (alexeykuzmin) [inline patches/common/chromium/can_create_window.patch]: Why do you want to change this? )

File: patches/common/chromium/can_create_window.patch [hunk 2]
```diff
@@ -23,13 +23,13 @@ index 1bf37e107a90..82ab04e1019d 100644
...
 +          break;
 +        }
 +        case storage::DataElement::TYPE_FILE_FILESYSTEM: {
 +          element.SetToFileSystemUrlRange(iter->filesystem_url,
-+                                          iter->offset, iter->length,
-+                                          iter->expected_modification_time);
++                                           iter->offset, iter->length,
++                                           iter->expected_modification_time);
```
Comment (alexeykuzmin) [inline patches/common/chromium/can_create_window.patch]: And these two? )

**Comment by alexeykuzmin** [comment]
```
./../../content/browser/frame_host/render_frame_host_impl.cc:2880:16: error: allocation of incomplete type 'content::ResourceRequestBody'
    body = new ResourceRequestBody;
               ^~~~~~~~~~~~~~~~~~~
```
`ResourceRequestBody` has been moved from `content::` to `network::`.
https://chromium-review.googlesource.com/864422

**Comment by alexeykuzmin** [comment]
```
./../../content/browser/frame_host/render_frame_host_impl.cc:2881:26: error: no member named 'DataElement' in namespace 'storage'
    std::vector<storage::DataElement> elements;
                ~~~~~~~~~^
```
`DataElement ` has been moved to `network::`.
https://chromium-review.googlesource.com/861334

File: patches/common/chromium/can_create_window.patch [hunk 3]
```diff
@@ -23,13 +23,13 @@ index 1bf37e107a90..82ab04e1019d 100644
 +        }
 +        case storage::DataElement::TYPE_FILE: {
 +          element.SetToFilePathRange(iter->path, iter->offset, iter->length,
-+                                     iter->expected_modification_time);
++                                      iter->expected_modification_time);
 +          break;
 +        }
 +        case storage::DataElement::TYPE_FILE_FILESYSTEM: {
```
Comment (alexeykuzmin) [inline patches/common/chromium/can_create_window.patch]: Some types have been removed as no longer used.
https://chromium-review.googlesource.com/877185

`TYPE_BYTES_DESCRIPTION`, `TYPE_DISK_CACHE_ENTRY`, `TYPE_FILE_FILESYSTEM`.

File: patches/common/chromium/can_create_window.patch [hunk 4]
```diff
@@ -172,49 +163,40 @@ index 09a26d8a7a6c..ffc823d49817 100644
...
++        case network::DataElement::TYPE_FILE: {
 +          ptr->path = element.path();
 +          ptr->offset = element.offset();
 +          ptr->length = element.length();
 +          ptr->expected_modification_time = element.expected_modification_time();
 +          break;
 +        }
-+        case storage::DataElement::TYPE_FILE_FILESYSTEM: {
-+          ptr->filesystem_url = element.filesystem_url();
++        case network::DataElement::TYPE_RAW_FILE: {
```
Comment (deepak1556) [inline patches/common/chromium/can_create_window.patch]: `network::DataElement::TYPE_RAW_FILE` should be ignored, we don't have data writer setup for it.

```
case network::DataElement::TYPE_RAW_FILE:
case network::DataElement::TYPE_DATA_PIPE:
case network::DataElement::TYPE_UNKNOWN:
        NOTREACHED();
        break;
```

**Pull request merged**